codalab/rest/bundles.py

```diff
@@ -1314,7 +1315,12 @@ def delete_bundles(uuids, force, recursive, data_only, dry_run):
             local.model.delete_bundles(relevant_uuids)
 
         # Update user statistics
-        local.model.update_user_disk_used(request.user.user_id)
+        # Just decrement the user disk used by the sum of sizes of bundles deleted
+        # OK, now let's add our change.
+        bundle_data_sizes = local.model.get_bundle_metadata(relevant_uuids, 'data_size')
```

Comment:
Also check the case where some bundles that are being deleted don't have data (e.g., a bundle which has had `cl rm -d ...` done on it)

Reply:
Checked that case with the new tests.